Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -2521,7 +2523,8 @@ def handle_incoming_user_unassignment(incident_data):
 
 def handle_incoming_closing_incident(incident_data):
     closing_entry = {}  # type: Dict
-    if incident_data.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR:
+    incident_status = incident_data.get('status')
+    if incident_status in XDR_RESOLVED_STATUS_TO_XSOAR:
```

Comment:
You can use walrus
```suggestion
    if incident_status := incident_data.get('status') in XDR_RESOLVED_STATUS_TO_XSOAR:
```

Reply:
Will fix that in the merged PR to master after customer approval it solves the problem